src/ui/components/results/verticalresultscomponent.js

```diff
@@ -191,16 +220,15 @@ export default class VerticalResultsComponent extends Component {
     this.resultsCount = data.resultsCount;
     this.verticalKey = data.verticalConfigId;
     this.resultsContext = data.resultsContext;
-    this.appliedQueryFilters = (data.appliedQueryFilters || [])
-      .filter(f => !this._config.hiddenFields.includes(f.fieldId));
+    this.nlpFilters = data.appliedQueryFilters || [];
     const searchState = data.searchState || SearchStates.PRE_SEARCH;
     const displayResultsIfExist = this._config.isUniversal ||
       this._displayAllResults ||
       data.resultsContext === ResultsContext.NORMAL;
+    this.appliedFilterNodes = this._getAppliedFilterNodes(this._convertNlpFiltersToFilterNodes(this.nlpFilters));
```

Comment:
Line break after '='

Reply:
Minor nit, but how about we have getAppliedFilterNodes be responsible for calling the nlpFilter conversion too? I think that could consolidate things nicely.

Reply:
Sure, updated!